test/suites/static_analysis.sh

```diff
@@ -24,8 +24,12 @@ test_static_analysis() {
       false
     fi
 
-    ## go vet
-    go vet ./...
+    ## go vet, if it exists
+    have_go_vet=1
+    go help vet || have_go_vet=0
```

Comment:
Can you >/dev/null 2>&1 ? Looks a bit odd in the testsuite output with the help showing up randomly in there :)